text_collector_examples/apcaccess.sh

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env bash
+#
+# Description: Expose metrics from apcaccess
+#
+# Author: Erwin Oegema <erwin.oegema@gmail.com>
+# Required tools:
+# - apcaccess
+# - bc
+# - awk
+# - cut
+
+set -ue
```

Comment:
I prefer to not use `set -e` to avoid bash silently exiting on errors. If you have places where errors may happen, you should handle them explicitly with `$?`.

I would also recommend `set -o pipefail` to avoid passing errors through pipes.